src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -45,14 +45,24 @@ private static X509Certificate2Collection GetCertificateCollection(string certif
             {
                 // On Windows, .Net Core applications should not import PFX files in parallel to avoid a known system-level race condition.
                 // This bug results in corrupting the X509Certificate2 certificate state.
-                Assert.True(m.WaitOne(MutexTimeout), "Cannot acquire the global certificate mutex.");
-
-                var certCollection = new X509Certificate2Collection();
-                certCollection.Import(Path.Combine(TestDataFolder, certificateFileName), CertificatePassword, X509KeyStorageFlags.DefaultKeySet);
+                try
```

Comment:
> new X509Certificate2(aCertFromAPfxWithoutPersistKeySet).Dispose()

Also, you're saying that if I do:
```C#
X509Certificate2 orig = ...;
X509Certificate2 clone = new X509Certificate2(orig);
orig.Dispose();
```
then `clone` is no longer usable?  That's... well.. 😦